### PR TITLE
[Fix] Fix typos and errors

### PR DESCRIPTION
**Pull request by Wuziyi616**
Fix a link error in README file and several typos in comments and unittest files.

### CODE REVIEW

**Review by THU17cyz — APPROVED** [review]

File: configs/votenet/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ We implement VoteNet and provide the result and checkpoints on ScanNet and SUNRG
 | :---------: | :-----: | :------: | :------------: | :----: |:----: | :------: |
 |    [PointNet++](./votenet_16x8_sunrgbd-3d-10class.py)     |  3x    |8.1||59.07|35.77|[model](https://download.openmmlab.com/mmdetection3d/v0.1.0_models/votenet/votenet_16x8_sunrgbd-3d-10class/votenet_16x8_sunrgbd-3d-10class_20200620_230238-4483c0c0.pth) &#124; [log](https://download.openmmlab.com/mmdetection3d/v0.1.0_models/votenet/votenet_16x8_sunrgbd-3d-10class/votenet_16x8_sunrgbd-3d-10class_20200620_230238.log.json)|
 
-**Notice**: If your current mmdetection3d version >= 0.6.0, and you are using the checkpoints downloaded from the above links or using checkpoints trained with mmdetection3d version < 0.6.0, the checkpoints have to be first converted via [tools/convert_votenet_checkpoints.py](../../tools/convert_votenet_checkpoints.py):
+**Notice**: If your current mmdetection3d version >= 0.6.0, and you are using the checkpoints downloaded from the above links or using checkpoints trained with mmdetection3d version < 0.6.0, the checkpoints have to be first converted via [tools/model_converters/convert_votenet_checkpoints.py](../../tools/model_converters/convert_votenet_checkpoints.py):
```
Comment (ZwwWayne) [inline configs/votenet/README.md:32]: Can we further check whether other places have such issues and fix them all in this PR?

**Comment by Wuziyi616** [comment]
Still looking for more link errors, just a moment.

File: configs/votenet/README.md (hunk 1 above)
Comment (Wuziyi616) [reply configs/votenet/README.md:32]: I have checked all the link errors for files under tools/ folder.

**Comment by Wuziyi616** [comment]
Checking link errors done, add curly brackets to latex citations in README files.

File: docs/useful_tools.md [hunk 2]
```diff
@@ -57,19 +57,23 @@ To see the SUNRGBD, ScanNet or KITTI points and detection results, you can run t
...
+Aftering running this command, plotted results **_\_points.obj and _**\_pred.ply files in `${SHOW_DIR}`.
 
 To see the points, detection results and ground truth of SUNRGBD, ScanNet or KITTI during evaluation time, you can run the following command
+
 ```bash
 python tools/test.py ${CONFIG_FILE} ${CKPT_PATH} --eval 'mAP' --options 'show=True' 'out_dir=${SHOW_DIR}'
 ```
-After running this command, you will obtain ***_points.ob, ***_pred.ply files and ***_gt.ply in `${SHOW_DIR}`. When `show` is enabled, [Open3D](http://www.open3d.org/) will be used to visualize the results online. You need to set `show=False` while running test in remote server withou GUI.
+
+After running this command, you will obtain **_\_points.ob, _**\_pred.ply files and \*\*\*\_gt.ply in `${SHOW_DIR}`. When `show` is enabled, [Open3D](http://www.open3d.org/) will be used to visualize the results online. You need to set `show=False` while running test in remote server withou GUI.
```
Comment (THU17cyz) [inline docs/useful_tools.md:68]: Is points.ob a typo?

Comment (Wuziyi616) [reply docs/useful_tools.md]: Indeed it is a typo. I have fixed it in the new commits.

**Review by Tai-Wang — APPROVED** [review]

**Pull request merged**